packages/manager/src/factories/firewalls.ts

```diff
@@ -58,9 +59,20 @@ export const firewallDeviceFactory = Factory.Sync.makeFactory<FirewallDevice>({
   updated: '2020-01-01',
 });
 
+const firewallTemplateSlugs: FirewallTemplateSlug[] = [
+  'akamai-non-prod',
+  'vpc',
+  'public',
+];
+
 export const firewallTemplateFactory = Factory.Sync.makeFactory<FirewallTemplate>(
   {
     rules: firewallRulesFactory.build(),
-    slug: Factory.each((i) => `template-${i}`),
+    slug: Factory.each((_) => {
+      const randomIndex = Math.floor(
+        Math.random() * firewallTemplateSlugs.length
+      );
+      return firewallTemplateSlugs[randomIndex];
+    }),
```

Comment:
Can we remove the randomness here and just use a static value?

 Our docs on Factories advise against it https://linode.github.io/manager/development-guide/09-mocking-data.html#factories

```suggestion
    slug: 'akamai-non-prod',
```